css/index.css

```diff
@@ -112,6 +141,119 @@ p {
     justify-content: center;
 }
 
```

Comment:
With each individual box, you only want to override what needs to change, in this case, the color.  The parent class can take care all of the other styling.